crates/ruff_python_formatter/src/expression/expr_dict.rs

```diff
@@ -69,7 +70,8 @@ impl FormatNodeRule<ExprDict> for FormatExprDict {
         let dangling = comments.dangling_comments(item);
 
         if values.is_empty() {
-            return empty_parenthesized_with_dangling_comments(text("{"), dangling, text("}"))
+            return empty_parenthesized("{", "}")
+                .with_dangling_comments(dangling)
```

Comment:
It seems all call paths have dangling comments. I would then change the API to `empty_parenthesized("{", dangling, "}")` as @konstin suggested.

Reply:
Hmm, ok, I will revert if you both agree. I find it weird that it has a totally different API than `parenthesized`.

Reply:
(I see the value in it though, now they're (appropriately) required.)